src/nplinker/pairedomics/podp_antismash_downloader.py

```diff
@@ -28,41 +34,86 @@
 
 class GenomeStatus:
 
-    def __init__(self, original_id, resolved_id, attempted=False, filename=""):
+    def __init__(self,
+                 original_id,
+                 resolved_refseq_id,
+                 attempted=False,
+                 filename=""):
         self.original_id = ';'.join(original_id.split(','))
-        self.resolved_id = None if resolved_id == 'None' else resolved_id
-        self.attempted = True if attempted == 'True' else False
+        self.resolved_refseq_id = None if resolved_refseq_id == 'None' else resolved_refseq_id
+        if ast.literal_eval(attempted):
+            self.attempted = True
+        else:
+            self.attempted = False
         self.filename = filename
 
     @classmethod
-    def from_csv(cls, original_id, resolved_id, attempted, filename):
-        return cls(original_id, resolved_id, attempted, filename)
+    def from_csv(cls, original_id, resolved_refseq_id, attempted, filename):
+        return cls(original_id, resolved_refseq_id, attempted, filename)
 
     def to_csv(self):
```

Comment:
```suggestion
    def to_csv(self, file: str | Pathlike, mode: 'w'):
```

`to_csv` method usually means writing the object to a csv file (like [this example](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_csv.html)), please implement the method with the given parameters